file-formats/sajc/README.md

```diff
@@ -1,5 +1,5 @@
 # SAJC File Format
 
 File | Cardinality | Definition | Schema | Example
-:--- | :---  | :--- | :--- | :---
-`<name>.sajc.json` | 1 | [`zif_aff_sajc_v1.intf.abap`](./type/zif_aff_sajc_v1.intf.abap) | [`sajc-v1.json`](./sajc-v1.json) | [`z_aff_example_sajc.sajc.json`](./examples/z_aff_example_sajc.sajc.json)
+:--- | :--- | :--- | :--- | :---
+`<name>.sajc.json` | 1 |  [`zif_aff_sajc_v1.intf.abap`](./type/zif_aff_sajc_v1.intf.abap)  | [`sajc-v1.json`](./sajc-v1.json) | [z_aff_example_sajc.sajc.json](./examples/z_aff_example_sajc.sajc.json)
```

Comment:
```suggestion
`<name>.sajc.json` | 1 | [`zif_aff_sajc_v1.intf.abap`](./type/zif_aff_sajc_v1.intf.abap) | [`sajc-v1.json`](./sajc-v1.json) | [`z_aff_example_sajc.sajc.json`](./examples/z_aff_example_sajc.sajc.json)
```

Reply:
I suggest to change it in this PR manually. 

The generator report must be fixed. See SAP/abap-file-formats-tools#300

Reply:
I changed it.